src/librbd/mirror/snapshot/CreatePrimaryRequest.cc

```diff
@@ -42,7 +42,7 @@ CreatePrimaryRequest<I>::CreatePrimaryRequest(
 
 template <typename I>
 void CreatePrimaryRequest<I>::send() {
-  if (!util::can_create_primary_snapshot(
+  if (m_group_snap_id.empty() && !util::can_create_primary_snapshot(
```

Comment:
This will skip the image level checks if it is part of a group. Please confirm that this will not cause issues.

Reply:
Please remove this change - it will affect multiple operations which create primary snaps.